src/segment-loader.js

```diff
@@ -2529,6 +2529,11 @@ export default class SegmentLoader extends videojs.EventTarget {
    * @private
    */
   handleAppendsDone_() {
+    // appendsdone can cause an abort
+    if (this.pendingSegment_) {
+      this.trigger('appendsdone');
```

Comment:
this can cause `this.pendingSegment_` to become `null` via an abort.

Reply:
So `pendingSegment_` be removed as part of `appendsdone`? Which will allow us to enter into the if statement below?

Reply:
Yes